### ISSUE TITLE
Error query column in table with JSON/JSONB

### ISSUE THREAD

**Issue by abusaidm** [Microsoft/vscode-postgresql#17]
- PGSQL Extension Version: 0.1.0
- VSCode Version:  1.32.3 (1.32.3)
- OS Version: Mac OS 10.14.2

Steps to Reproduce:

1. Connect to a postgres DB
2. Run Select * on any table that has a JSON/JSONB type field

```
[10:15:32 AM]	Started executing query at Line 2
	
	Commands completed successfully 
	
	Unhandled exception while executing query: 'dict' object has no attribute 'encode' 
	
	Total execution time: 00:00:01.625
```

Thanks for the new plugin

**Comment by avrahamkam** [comment]
I've confirmed this problem.

**Comment by fleetside72** [comment]
I have the same experience except that `select '{"key":"val"}'::json` is OK, but casting to jsonb does not work (`select '{"key":"val"}'::jsonb`)

**Comment by akamath46297** [comment]
There is a PR https://github.com/Microsoft/pgtoolsservice/pull/208 out for the fix. We will validate and get this in.

**Comment by akamath46297** [comment]
This is now fixed with [Release 0.2.0](https://github.com/Microsoft/vscode-postgresql/releases/tag/v0.2.0)

**Comment by akamath46297** [comment]
@abusaidm , @avrahamkam , @fleetside72 . FYI..
